std/protocols/bus.asm

```diff
@@ -131,4 +206,17 @@ let bus_send: expr, expr[], expr -> () = constr |id, payload, multiplicity| {
 /// Convenience function for bus interaction to receive columns
 let bus_receive: expr, expr[], expr, expr -> () = constr |id, payload, multiplicity, latch| {
     bus_interaction(id, payload, -multiplicity, latch);
-};
\ No newline at end of file
+};
+
+/// Convenience function for batching two bus sends.
+let bus_multi_send: expr, expr[], expr, expr, expr[], expr -> () = constr |id_0, payload_0, multiplicity_0, id_1, payload_1, multiplicity_1| {
+    // For bus sends, the multiplicity always equals the latch
+    bus_multi_interaction(id_0, payload_0, multiplicity_0, multiplicity_0, id_1, payload_1, multiplicity_1, multiplicity_1);
+};
+
+/// Convenience function for batching two bus receives.
+/// In practice, can also batch one bus send and one bus receive, but requires knowledge of this function and careful configuration of input parameters.
+/// E.g. sending negative multiplicity and multiplicity for "multiplicity" and "latch" parameters for bus sends.
```

Comment:
👍

Reply:
Maybe eventually it makes sense to to only have one function `bus_interactions` that receives a list of elements of the type:
```rust
enum BusInteraction {
  Send(expr, expr[], expr, expr),
  Receive(expr, expr[], expr, expr),
}
```

Then the API is about as intuitive as now, but it gives us more flexibility and we can get rid of the send / receive wrappers.